cmd/test.go

```diff
@@ -107,6 +107,8 @@ type Test struct {
 	logger *zap.Logger
 }
 
+var deleteTestReport bool
```

Comment:
Take flag name as `generateTestReport` which should be true by default.

Reply:
Changed flag name and set default value to true

The implementation is now as follows:

![Uploading image.png…]()